### DISCUSSION TITLE
Use static point cloud as map

### DISCUSSION THREAD

**Discussion by anchalsinha** [stella-cv/stella_vslam#251] [Q&A] [answered]
Would it be possible to generate a map db from a dense point cloud (containing XYZ RGB data). I have a point cloud map of a scene I want to localize by monocular camera within, and wanted to see if I could make use of OpenVSLAM's localization tools to do that. 

I would assume I would need to detect the "landmarks", but it seems like landmarks and keyframes are detected from a camera moving through the scene with the SLAM algorithms.
Any advice if performing localization on a pre-built point cloud would be possible using OpenVSLAM's tools?

**Comment by ymd-stella** [comment, marked as answer]
Not possible.
